mlflow_databricks_artifacts/utils/string_utils.py

```diff
@@ -0,0 +1,12 @@
+def strip_prefix(original, prefix):
```

Comment:
In accordance with https://github.com/dbczumar/mlflow-databricks-artifacts/pull/1/files#r486553651, these utilities (which are private APIs and have behavioral differences across versions of MLflow) have been copied out of MLflow 1.11 for consistency.